config/config.go

```diff
@@ -115,3 +116,23 @@ type Cert struct {
 	CertFile string `envconfig:"CERT_FILE"`
 	KeyFile  string `envconfig:"KEY_FILE"`
 }
+
+type Metrics struct {
+	Type     string
+	Log      LogMetrics
+	Syslog   SyslogMetrics
+	Interval time.Duration `default:"10s"`
+}
+
+type LogMetrics struct {
```

Comment:
__[golint]__
* [golint] exported type LogMetrics should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->